### PR TITLE
Version flag

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
**Description of the Pull Request (PR):**

Add version sub cmd

**Checkoff for all PRs:**

- [x] I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [ ] I have added changes to the [CHANGELOG](https://github.com/singularityware/singularity/blob/development/CHANGELOG.md) and and documentation updates to the [singularityware](https://www.github.com/singularityware/singularityware.github.io) documentation base.
- [x] I have tested this PR locally with a `make test`
- [x] This PR is NOT against the project's master branch
- [x] I have added myself as a contributor to the [contributors's file](https://github.com/singularityware/singularity/blob/master/CONTRIBUTORS.md)
- [x] This PR is ready for review and/or merge

Attn: @singularityware-admin

### CODE REVIEW

File: src/cmd/singularity/cli/verify.go [hunk 1]
```diff
@@ -1,10 +1,7 @@
-/*
-  Copyright (c) 2018, Sylabs, Inc. All rights reserved.
-
-  This software is licensed under a 3-clause BSD license.  Please
-  consult LICENSE file distributed with the sources of this project regarding
-  your rights to use or distribute this software.
-*/
+//   Copyright (c) 2018, Sylabs, Inc. All rights reserved.
+//   This software is licensed under a 3-clause BSD license.  Please
+//   consult LICENSE file distributed with the sources of this project regarding
```
Comment (jmstover) [inline src/cmd/singularity/cli/verify.go]: Change this to be:
`consult LICENSE.md file distributed with the sources`

**Review by jmstover — CHANGES_REQUESTED** [review]
Change the LICENSE file mentioned to be correct

**Review by jmstover — DISMISSED** [review]

**Review by jmstover — APPROVED** [review]

**Comment by tri-adam** [comment]
I'd prefer if we didn't have the branch name in there - a commit is more specific, and the branch(es) can be derived from that.

What I _would_ like to see is the use of proper tagging, along with the addition of a `dirty`, which Git has built-in support for. What about using this:

```sh
$ git describe --long --dirty
2.4.2-1177-g2e7707e7
```

The `-dirty` flag will allow us to tell if a user has local modifications on top of the commit they are on, which is very useful when debugging. And the middle field (`1177`) allows us to quickly order builds by which is "newer".

**Comment by GodloveD** [comment]
I think I support these changes, but I need a bit more info too.  What is the middle field and how does it help?

**Comment by tri-adam** [comment]
OK so `2.4.2-1177-g2e7707e7` would mean the most recent tag is `2.4.2`, there have been `1177` commits since that tag, and the most recent commit is `g2e7707e7`.

There's a more detailed explanation at https://git-scm.com/docs/git-describe#_examples.

**Comment by GodloveD** [comment]
I support these changes. 😄

**Pull request merged**